guides/source/action_cable_overview.md

```diff
@@ -120,6 +120,17 @@ by this same current user, you're also ensuring that you can later retrieve all
 connections by a given user (and potentially disconnect them all if the user is deleted
 or unauthorized).
 
+If you use Devise for authenticaion, you can get `current_user` from warden:
```

Comment:
Can we remove the devise part? We don't talk about non-default gems in the guides.

Reply:
Of course. Done. The thing is that devise is already mentioned in the bottom of this guide (section "Notes"). So, a followup questions:
 - Should I remove it also from there?
 - Should I move my sentence to the "Notes" section (I personally find it more likely to be useful if it stays where it is).

Reply:
Moving to the notes make sense, but talking about Devise in the regular flow is something we want to avoid.